main.tex

```diff
@@ -316,54 +318,46 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
 %
-If mosquitoes from different geographical locations or species carry the same resistance allele on identical or near-identical genetic backgrounds, this implies that the allele has been spread between mosquito populations by the movement and interbreeding of mosquitoes.
+In our initial report of the Ag1000G phase 1 resource \cite{Ag1000gConsortium2017}, we used 1710 biallelic SNPs from within the 73.5 kbp \textit{Vgsc} gene (1607 intronic, 103 exonic) to compute the number of SNP differences between all pairs of 1530 haplotypes derived from 765 wild-caught mosquitoes.
+%
+We then used pairwise genetic distances to perform hierarchical clustering, and found that haplotypes carrying resistance alleles in codon 995 were grouped into 10 distinct clusters, each with near-identical haplotypes.
 %
-Conversely, if the same resistance allele is found on different genetic backgrounds in different mosquito populations, this provides evidence that the allele has emerged independently in each population, either because of multiple mutational events since the introduction of pyrethroids, or because the allele was segregating at low frequency prior to the introduction of pyrethroids.
+Five of these clusters contained haplotypes carrying the \texttt{L995F} allele (labelled F1-F5), and a further five clusters contained haplotypes carrying \texttt{L995S} (labelled S1-S5).
 %%
 
 %%
 %
-In our initial report of the Ag1000G phase 1 resource \cite{Ag1000gConsortium2017}, we used 1710 biallelic SNPs from within the 73.5 kbp \textit{Vgsc} gene (1607 intronic, 103 exonic) to compute the number of SNP differences between all pairs of 1530 haplotypes derived from 765 wild-caught mosquitoes.
-%
-We then used pairwise genetic distances to perform hierarchical clustering, and found that haplotypes carrying resistance alleles in codon \texttt{995} were grouped into 10 distinct clusters with near-identical haplotypes.
+To further investigate genetic backgrounds carrying resistance alleles in codon 995, we used the Ag1000G haplotype data to construct median-joining networks \cite{Bandelt1999} (Figure \ref{fig:networks}).
 %
-Five of these clusters carried the \texttt{L995F} allele (labelled F1-F5), and a further five clusters carried \texttt{L995S} (labelled S1-S5).
-%
-As an alternative approach, we used the same haplotype data to construct median-joining networks (Figure \ref{fig:networks}).
-%
-The network analysis allows for the reconstruction and placement of intermediate haplotypes that may not be observed in the data.
+The network analysis improves on hierarchical clustering by allowing for the reconstruction and placement of intermediate haplotypes that may not be observed in the data.
 %
 It also allows for non-hierarchical relationships between haplotypes, which may arise if recombination events have occured between haplotypes.
 %
-Furthermore, the networks allow relationships among closely-related haplotypes to be discerned.
```

Comment:
This seemed unnecessary so cut.

Reply:
The "improves on" part was requested to be removed in an earlier draft by (iirc) Martin as it suggests the previous analysis was inferior and a reviewer could ask "why did we do it then?". The "alternative approach" was what was suggested instead.

Reply:
I know, but as I was reshuffling a bit, it just didn't read as well. The network analysis is better than hierarchical clustering, for the reasons we give. I don't think we need to hide that. If a reviewer wants to question why we did hierarchical clustering in a previous paper, that would be a comment on the previous paper, not this one.

Reply:
Agreed, I'm happy with this version.